src/components/astra-editor/plugins/prompt-dialog.ts

```diff
@@ -0,0 +1,318 @@
+import { css, html, LitElement, nothing, type PropertyValues } from 'lit'
+import { customElement, property, state } from 'lit/decorators.js'
+import { createRef, ref, type Ref } from 'lit/directives/ref.js'
+import { unsafeHTML } from 'lit/directives/unsafe-html.js'
+
+const MINIMAL_HEIGHT = 40
+
+const LoadingSvg = `<svg class='spin' xmlns="http://www.w3.org/2000/svg" width="13" height="13" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-loader">
```

Comment:
I've been adding icons here https://github.com/outerbase/astra-ui/tree/main/src/icons if you want to mimic the syntax/style so they're all consistent?

Reply:
I will revisit this icon when I got chance to work on it again